css/projects.css

```diff
@@ -0,0 +1,107 @@
+
+/* Works Section Styling Starts Here */
+
+.works-card {
+  color: #172b4d;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1em;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+  margin: 8em 7% 0 7%;
+  padding: 1em 1em 2em 1em;
+}
```

Comment:
![image](https://user-images.githubusercontent.com/60083437/235312626-9d573fa3-146f-44d6-8102-521a438f2024.png)

- [ ] I have marked `4` problems in the above picture. All of the `4` problems are related to `extra gap/margin/padding` between the contents. Kindly decrease the gaps according to the given guideline.